R/bayesplot-colors.R

```diff
@@ -259,6 +314,7 @@ get_color <- function(levels) {
   color_vals <- color_scheme_get()[levels]
   unlist(color_vals, use.names = FALSE)
 }
+
 full_level_name <- function(x) {
```

Comment:
I'm adding a commit where I refactor this to not use a `switch()` but use the named-vector look-up table trick to get vectorization for free. adv-r.had.co.nz/Subsetting.html#lookup-tables